app/lib/activitypub/parser/media_attachment_parser.rb

```diff
@@ -9,9 +9,9 @@ def initialize(json)
 
   # @param [MediaAttachment] previous_record
   def significantly_changes?(previous_record)
-    remote_url != previous_record.remote_url ||
-      thumbnail_remote_url != previous_record.thumbnail_remote_url ||
-      description != previous_record.description
+    remote_url.presence != previous_record.remote_url.presence ||
+      thumbnail_remote_url.presence != previous_record.thumbnail_remote_url.presence ||
+      description.presence != previous_record.description.presence
```

Comment:
It won't hurt, but unlikely to fix any actual issue, as none of the spurious edits I've seen have been caused by a `nil` / empty string discrepancy, but by thumbnails being added.

Reply:
Actually, it seems we were both mistaken, and the issue isn't caused by `thumbnail_remote_url` at all, but by the “description” being processed in a `before_create` hook in `MediaAttachment`